ClientLib/src/main/java/com/o3dr/android/client/apis/drone/DroneStateApi.java

```diff
@@ -21,9 +22,10 @@
      *
      * @param arm true to arm, false to disarm.
      */
-    public static void arm(Drone drone, boolean arm) {
+    public static void arm(Drone drone, boolean arm, boolean emergencyDisarm) {
```

Comment:
To avoid breaking the api, keep the original method signature and add an overload with the new method signature. The original one should just be a delegate to the new one with the `emergencyDisarm` argument set to false.